composer/trainer/trainer.py

```diff
@@ -419,6 +419,11 @@ class Trainer:
             This setting will end each epoch early to avoid additional training that will not be profiled.
 
             This parameter is ignored if ``train_dataloader`` is not specified.
+        spin_dataloaders (bool, optional): If ``True``, dataloaders will be spun up to the current timestamp.
+            This ensures dataloaders continue from the same batch when resuming training. (default: ``True``)
+
+            .. note:: Spinning dataloaders can be potentially very slow. If resuming from a checkpoint where
```

Comment:
This note is a bit unclear to me. As is, it says that if you don't care about determinism, you're ok setting spin to False. This is somewhat incomplete information. Because if you're data order _is_ deterministic, you're not just not getting determinism, you're just only training on the first X samples (and repeating them however many times you restart). I'd rewrite to explain this a bit more thoroughly.